x-pack/plugins/alerting/server/config.ts

```diff
@@ -22,8 +22,16 @@ export const configSchema = schema.object({
   }),
   defaultRuleTaskTimeout: schema.string({ validate: validateDurationSchema, defaultValue: '5m' }),
   cancelAlertsOnRuleTimeout: schema.boolean({ defaultValue: true }),
-  minimumScheduleInterval: schema.string({ validate: validateDurationSchema, defaultValue: '1m' }),
+  rules: schema.object({
```

Comment:
Moved from top level config to under `xpack.alerting.rules.*` to follow the schema suggested in https://github.com/elastic/kibana/pull/126902. I did not move it under `execution` since it is not only related to execution, but I am open to feedback on this.

Reply:
I would define it like: `xpack.alerting.rules.execution.interval.min`
But IMHO this is ok too :)

Reply:
I'm +1 to leaving it out of execution. In the future, if ever we wanted to add a config that applies to all new and existing rule intervals, it could go under execution. We would just have to make it clear what each config would be for.